client.py

```diff
@@ -34,6 +23,67 @@
         if iteration_counter == 10:
             iteration_counter = 0
             print("Map size:", my_map.size())
+
+
+def sql_example(hz_client):
+    print("Creating a mapping...")
+    # See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
+    mapping_query = "CREATE MAPPING cities TYPE IMap OPTIONS ('keyFormat'='varchar','valueFormat'='varchar')"
+    hz_client.sql.execute(mapping_query)
+    print("The mapping has been created successfully.")
+    print("--------------------")
+
+    print("Inserting data via SQL...")
+    insert_query = """
+    INSERT INTO cities VALUES
+    ('Australia','Canberra'),
+    ('Croatia','Zagreb'),
+    ('Czech Republic','Prague'),
+    ('England','London'),
+    ('Turkey','Ankara'),
+    ('United States','Washington, DC');
+    """
+    hz_client.sql.execute(insert_query)
+    print("The data has been inserted successfully.")
+    print("--------------------")
+
+    print("Retrieving all the data via SQL...")
+    result = hz_client.sql.execute("SELECT * FROM cities").result()
+    for row in result:
+        country = row.get_object_with_index(0)
+        city = row.get_object_with_index(1)
+        print("%s - %s" % (country, city))
+    print("--------------------")
+
+    print("Retrieving a city name via SQL...")
+    result = hz_client.sql.execute("SELECT __key, this FROM cities WHERE __key = ?", "United States").result()
+    for row in result:
+        country = row.get_object("__key")
+        city = row.get_object("this")
+        print("Country name: %s; City name: %s" % (country, city))
+        print("--------------------")
+        exit(0)
+
+
+logging.basicConfig(level=logging.INFO)
+HazelcastCloudDiscovery._CLOUD_URL_BASE = "YOUR_DISCOVERY_URL"
+client = hazelcast.HazelcastClient(
+    cluster_name="YOUR_CLUSTER_NAME",
+    cloud_discovery_token="YOUR_CLUSTER_DISCOVERY_TOKEN",
+    statistics_enabled=True,
+)
+
+sample_map = client.get_map("map").blocking()
+sample_map.put("key", "value")
+
+if sample_map.get("key") == "value":
+    print("Successfully connected!")
+
+    # the 'map_example' is an example with an infinite loop inside, so if you'd like to try other examples,
+    # don't forget to comment out the following line
+    map_example(sample_map)
+
+    # sql_example(client)
 else:
     client.shutdown()
     raise Exception("Connection failed, check your configuration.")
```

Comment:
```suggestion
logging.basicConfig(level=logging.INFO)
HazelcastCloudDiscovery._CLOUD_URL_BASE = "YOUR_DISCOVERY_URL"
client = hazelcast.HazelcastClient(
    cluster_name="YOUR_CLUSTER_NAME",
    cloud_discovery_token="YOUR_CLUSTER_DISCOVERY_TOKEN",
    statistics_enabled=True,
)
print("Successfully connected!")

sample_map = client.get_map("map").blocking()
sample_map.put("key", "value")

# the 'map_example' is an example with an infinite loop inside, so if you'd like to try other examples,
# don't forget to comment out the following line
map_example(sample_map)

# sql_example(client)

client.shutdown()
```

I would suggest changing this part as if there is a connection problem, client construction will fail, so there is no need to check the value of a key to determine whether we are connected or not

Reply:
Got it, thanks. In that case, we don't need the line `sample_map.put("key", "value")`.